bin/admin/bootstrap-cluster.sh

```diff
@@ -122,14 +201,9 @@ case $node_type in
   *)
   # unknown node_type
   echo "unknown node_type"
-  echo $usage
+  printf "$usage"
   exit
   ;;
 esac
 
-if [ -n "$help" ]; then
```

Comment:
If there is a help option, show the detailed usage description and stop this script immediately, so I don't need to wait for the rest of this script is run.